go/parquet/internal/encoding/boolean_decoder.go

```diff
@@ -37,48 +37,56 @@ func (PlainBooleanDecoder) Type() parquet.Type {
 	return parquet.Types.Boolean
 }
 
+func (dec *PlainBooleanDecoder) SetData(nvals int, data []byte) error {
+	dec.decoder.SetData(nvals, data)
+	dec.bitOffset = 0
+	return nil
+}
```

Comment:
Shouldn't ignore the error?

```suggestion
	if err := dec.decoder.SetData(nvals, data); err != nil {
		return err
	}
	dec.bitOffset = 0
	return nil
}
```

Reply:
+1